pom.xml

```diff
@@ -114,7 +114,6 @@
     <lombok.version>1.18.22</lombok.version>
     <snakeyaml.version>1.28</snakeyaml.version>
     <bouncycastle.version>1.70</bouncycastle.version>
-    <commons-codec.version>1.15</commons-codec.version>
     <commons-compress.version>1.21</commons-compress.version>
```

Comment:
We might need to add some exclusions because I think that other commons dependencies are pulling `commons-codec` as well, though I guess it shouldn't be an issue.

Reply:
Checking dependency tree

Reply:
Dependency tree doesn't show any transitivity